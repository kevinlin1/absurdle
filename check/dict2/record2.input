glue
boom
spew
clue
ibex
