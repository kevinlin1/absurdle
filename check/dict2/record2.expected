9 words
> glue
: ⬜⬜⬜🟨

2 words
> boom
: 🟨⬜⬜⬜

1 words
> spew
: ⬜⬜🟩⬜

1 words
> clue
: ⬜⬜⬜🟨

1 words
> ibex
: 🟩🟩🟩🟩

Absurdle 5/∞

⬜⬜⬜🟨
🟨⬜⬜⬜
⬜⬜🟩⬜
⬜⬜⬜🟨
🟩🟩🟩🟩
